trigger/task_jira.go

```diff
@@ -315,8 +315,8 @@ func (j *jiraBuilder) makeCustomFields(customFields []evergreen.JIRANotification
 
 // historyURL provides a full URL to the test's task history page.
 func historyURL(t *task.Task, testName, uiRoot string) string {
-	return fmt.Sprintf("%v/task_history/%v/%v#%v=fail",
-		uiRoot, url.PathEscape(t.Project), url.PathEscape(t.Id), url.QueryEscape(testName))
+	return fmt.Sprintf("%s/task_history/%s/%s?revision=%s#/%s=fail",
```

Comment:
Question: what is the / after # for?

Reply:
The task history page uses it as a delimiter: https://github.com/evergreen-ci/evergreen/blob/master/public/static/js/task_history.js#L13